modules/nw-dns-operator.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+// * networking/dns/dns-operator.adoc
+
+[id="dns-operator-{context}"]
+= DNS Operator
+
+The DNS Operator implements the `dns` API from the `operator.openshift.io` API
+group. The Operator manages a DNS Service for the cluster, and configures
+`kubelets` to tell individual containers to use the Service IP of the CoreDNS
+DaemonSet for name resolution.
```

Comment:
This sentence feels a little odd, and I think it's the way it mixes abstract description ("manages a DNS Service") with concrete details ("configures kubelets [...] to use the Service IP of the CoreDNS Daemonset [...]").  Since you have already established the abstract, high-level purpose of the operator in `networking/dns/dns-operator.adoc`, maybe you can stick to the concrete details here: "The operator deploys CoreDNS using a `DaemonSet`, creates a `Service` for the `DaemonSet`, and configures the Kubelet to tell pods to use the CoreDNS service IP for name resolution."  What do you think?

Reply:
Thanks for reviewing the docs. I agree with your feedback and will update the content.